### DISCUSSION TITLE
Reviews

### DISCUSSION THREAD

**Discussion by hermann-san** [piratenpanda/veggiekarte#131] [Ideas]
Hi,
there is a mention of lib.reviews on the landing page of veggiekarte.de
However, lib.reviews  is more or less a dead project. There is no further development happening.
There is a new Open Source project that is actively used and has some nice features, including images, reviews and the use of OSM.
It is mangrove.reviews and one can also review restaurants.  [MapComplete](https://mapcomplete.osm.be/food.html?z=17&lat=48.77377&lon=9.17809&language=de&filter-open_now=&filter-reservation=&filter-vegetarian=&filter-vegan=true&filter-halal=&filter-has_organic=&filter-accepts_cash=&filter-accepts_cards=#) also uses Mangrove.reviews for reviews
It would be nice if Veggiekarte could also use Mangrove.review

**Comment by KristjanESPERANTO** [comment]
Hi @hermann-san, I think this is a good idea! Can you create a PR?

**Reply by hermann-san** [reply]
Hi @KristjanESPERANTO , unfortunately I'm not a dev,.. sorry.
